xlators/mgmt/glusterd/src/glusterd.c

```diff
@@ -1485,15 +1485,15 @@ init(xlator_t *this)
     if (len < 0 || len >= PATH_MAX)
         exit(2);
 
-    dir_data = dict_get(this->options, "cluster-test-mode");
+    dir_data = dict_get(this->options, "logging-directory");
```

Comment:
Changing the name of an option is not backward compatible. We should support both names and, if possible, show a deprecation warning when the old name is used. After some releases we can remove the deprecated name.

Reply:
Ack, will update the code.